### ISSUE TITLE
Inconsistency in branding -> (colors, logo_url) between a0deploy versions and formats

### ISSUE THREAD

**Issue by joshw** [auth0/auth0-deploy-cli#648]
### Description

We have been using version 7.9.0, and when I went to update to 7.14.3, i noticed some changes in the yaml format output of export.   I tried several versions, and it seems like this changed between 7.10.0 and 7.11.0.

Specifically, in version 7.11.0 and higher, the "branding" section is missing the colors and logo_url properties in the yaml output format.  They do seem to be present in the directory format.

I think these fields were accidentally removed from the yaml format, but even if this change was deliberate, it seems like the two output formats should agree.

> Provide a clear and concise description of the issue, including what you expected to happen.

### Reproduction

In 7.10.0, in the yaml format, tenant.yaml contains
```yaml
branding:
  colors:
    primary: '#f55e05'
    page_background: '#000000'
  logo_url: <my url>
  templates: []
```
and in directory mode, branding/branding.json contains
```json
{
  "colors": {
    "primary": "#f55e05",
    "page_background": "#000000"
  },
  "logo_url": "<my url>"
}
```

In newer versions, however, this is different.    In 7.14.3, I get this instead:

yaml mode:
```yaml
branding:
  templates: []
```
directory mode:
```json
{
  "colors": {
    "primary": "#f55e05",
    "page_background": "#000000"
  },
  "logo_url": "<url>"
}
```

**Comment by willvedd** [comment]
Yep, this is certainly a bug. I've identified that somewhere between 7.9.0 and current, the non-template branding settings are omitted for the YAML configuration format. Will look to address this very soon.

**Comment by willvedd** [comment]
Thanks for opening this issue. It has been fixed with [7.15.1](https://github.com/auth0/auth0-deploy-cli/releases/tag/v7.15.1).
